packages/manager/src/features/PlacementGroups/constants.ts

```diff
@@ -8,3 +8,5 @@ export const MAX_NUMBER_OF_LINODES_IN_PLACEMENT_GROUP_MESSAGE =
 
 export const PLACEMENT_GROUP_LINODES_ERROR_MESSAGE =
   'There was an error loading Linodes for this Placement Group.';
+
+export const PLACEMENT_GROUP_TOOLTIP_TEXT = `The Affinity Type and Region determine the maximum number of VMs per group`;
```

Comment:
```suggestion
export const PLACEMENT_GROUP_TOOLTIP_TEXT = `The Affinity Type and Region determine the maximum number of VMs per group.`;
```
Nit: sentence needs a period.

Reply:
Also: are we using VMs in copy, but referring to them as Linodes in code, or should "VMs" also be "Linodes" here?

Reply:
There's a lot of copy that is still being worked, I will reach out to our technical writer about this and make any necessary changes in the a separate PR.